ci/Dockerfile

```diff
@@ -51,6 +51,7 @@ RUN apt update -yq && apt install -yq software-properties-common \
     libxext6 libxrender1 libxkbcommon-dev libxkbcommon-x11-dev libxcomposite1 libxtst6 \
     libxrandr2 libxcursor1 libxi6 libxcb-randr0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 \
     libxcb-render-util0 libxcb-shape0 libxcb-render0 libxcb-xinerama0 \
+    unixodbc-dev \
```

Comment:
Move them to where the rest of dev packages are:
![image](https://user-images.githubusercontent.com/2212681/223277624-849377b0-712d-44d7-b8d7-5d31b1527567.png)